tests/unit/memefficiency.tcl

```diff
@@ -577,4 +583,63 @@ start_server {tags {"defrag external:skip"} overrides {appendonly yes auto-aof-r
         }
     }
 }
+
+start_cluster 1 0 {tags {"defrag external:skip"}} {
+    if {[string match {*jemalloc*} [s mem_allocator]] && [r debug mallctl arenas.page] <= 8192} {
+        test "Active defrag slot_to_keys crash when calling flush async in cluster mode" {
```

Comment:
We could either name the test as it doesn't crash like `test "Active defrag doesn't crash when calling flush async in cluster mode"` or just simplify it.

```suggestion
        test "Active defrag with flush async performed in cluster mode" {
```